compiler/resolution/functionResolution.cpp

```diff
@@ -1706,6 +1895,9 @@ handleSymExprInExpandVarArgs(FnSymbol*  workingFn,
         }
       }
 
```

Comment:
It's hard to tell in a browser but I think the indentation is off here.  My guess is this is another
instance in which you've inserted a conditional but hesitated to re-indent correctly.